processor/src/main/java/com/linecorp/decaton/processor/runtime/TracingProvider.java

```diff
@@ -58,6 +58,8 @@
          * Otherwise, this will be invoked by the thread that calls {@link DeferredCompletion#complete()}.
          */
         void processingCompletion();
+
+        TraceHandle childFor(DecatonProcessor<?> processor);
```

Comment:
btw I should've caught this in last review, but I think this class should be put under `decaton.processor` package rather than `decadton.processor.runtime` as in the former is the place to put public interfaces that's supposed to be implemented by users.

Reply:
> The argument now can be String name? Any extra information you imagine it to obtain from processor instance?

I was thinking some implementations might want to cast / use reflection to access some extra information from the processor instance (e.g. extra tags or metrics).